server/controllers/index.js

```diff
@@ -0,0 +1,95 @@
+const models = require('../models/index.js');
```

Comment:
this is interesting. I'm not sure I've done much with controller files. Definitely curious to learn more about this sometime